crowbar_framework/app/models/pacemaker_service.rb

```diff
@@ -33,14 +33,14 @@ def role_constraints
           "unique" => false,
           "count" => 32,
           "platform" => {
-            "suse" => "/11\.[3-9]/"
+            "suse" => "/.*/"
```

Comment:
You enable it only for SUSE, but within the cookbooks you've got checks if it's not SUSE?